pkg/types/nutanix/client.go

```diff
@@ -0,0 +1,35 @@
+package nutanix
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	nutanixClient "github.com/terraform-providers/terraform-provider-nutanix/client"
+	nutanixClientV3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	"k8s.io/klog"
+)
+
+
+func CreateNutanixClient(ctx context.Context, prismCentral, port, username, password string, insecure bool) (*nutanixClientV3.Client, error) {
```

Comment:
This does not seem appropriate to include in this PR, given that the scope of this PR is limited to adding the platform types.